CRM/Contact/BAO/Query.php

```diff
@@ -6162,6 +6158,29 @@ public function convertGroupIDStringToLabelString(&$dao, $val) {
     // Note that groups that the user does not have permission to will be excluded (good).
     $groups = array_intersect_key($allGroups, array_flip($groupIDs));
     return implode(', ', $groups);
+
+  }
+
+  /**
+   * Set the qill and where properties for a field.
```

Comment:
I've never really figured it out what 'qill' means - but it refers to the text label of the search